djangocms_versioning_filer/plugins/audio/models.py

```diff
@@ -6,15 +6,19 @@
 from cms.models import CMSPlugin
 
 from djangocms_attributes_field.fields import AttributesField
-from djangocms_audio.models import AudioFolder, AudioTrack, get_extensions
+from djangocms_audio.models import AudioFolder, AudioTrack
 
 from djangocms_versioning_filer.fields import FileGrouperField
 from djangocms_versioning_filer.models import (
     get_files_distinct_grouper_queryset,
 )
 
 
-ALLOWED_EXTENSIONS = get_extensions()
+try:
```

Comment:
@jonathan-s Hate to be a pain, can we add a comment detailing which versions of djangocms_audio forced this change, if we detail version numbers one day in the future they will be redundant and this can be cleaned out. Th same for video, added the same comment to that too.